crates/mun_syntax/src/parsing/lexer/numbers.rs

```diff
@@ -14,7 +14,7 @@ pub(crate) fn scan_number(c: char, cursor: &mut Cursor) -> SyntaxKind {
                 scan_digits(cursor, true);
             }
             '0'..='9' | '_' | '.' | 'e' | 'E' => {
-                scan_digits(cursor, true);
+                scan_digits(cursor, false);
```

Comment:
Does an invocation of this call exist where `allow_hex == true`? If not, that argument can be removed.

Reply:
Yes in the part were a hex is part. One match case above this one.